server/src/routes/api/v1/buildsReviewsRouter.js

```diff
@@ -0,0 +1,30 @@
+import express from "express";
+import { Build, Review } from "../../../models/index.js"
+import objection from "objection"
+const { ValidationError } = objection
+import cleanBuildForm from "../../../services/cleanBuildForm.js";
+
+const buildsReviewsRouter = new express.Router({mergeParams: true})
+
+buildsReviewsRouter.post("/", async (req, res) => {
+    
+    const currentlyLoggedInUser = req.user
+    const submittedReview = req.body
+
+    submittedReview.buildId = req.params.id
+    submittedReview.userId = currentlyLoggedInUser.id
+
+    try {
+        const cleanedFormData = cleanBuildForm(submittedReview)
+        const reviewToAdd = await Review.query().insert(cleanedFormData)
+        return res.status(201).json({ reviewToAdd })
+    } catch (error) {
+        console.log(error)
+        if(error instanceof ValidationError){
+            return res.status(422).json({errors: error.data})
```

Comment:
space pad inside JS objects (just like line 26)
https://github.com/airbnb/javascript?tab=readme-ov-file#whitespace--in-braces

Reply:
done